Lab 2/README.md

```diff
@@ -71,23 +71,23 @@ A *replica* is how Kubernetes accomplishes scaling out a deployment. A replica i
 
 Kubernetes allows you to use a rollout to update an app deployment with a new Docker image.  This allows you to easily update the running image and also allows you to easily undo a rollout, if a problem is discovered after deployment.
 
-In the previous lab, we created an image with a `v1` tag. Let's make a `v2` tag with new content. This lab also contains a `Dockerfile`. Let's build and push it up to our image registry.
+In the previous lab, we created an image with a version `1` tag. Let's make a version `2` tag with new content. This lab also contains a `Dockerfile`. Let's build and push it up to our image registry.
```

Comment:
tags aren't versions, they're just strings.

Reply:
good point. I'll update. We can also keep v1 and v2, but if we do then we need to get dw course to change.